applications/CoSimulationApplication/python_scripts/custom_convergence_accelerators/convergence_accelerator_mvqn.py

```diff
@@ -0,0 +1,88 @@
+## @module iqnils
+# This module contains the class MVQN
+# Author: Wei He
+# Date: Feb. 20, 2017
+
+import numpy as np 
```

Comment:
this will only work in serial.

Reply:
yes, this algorithm does not yet work in MPI